config/locales/en.yml

```diff
@@ -1133,10 +1133,17 @@ en:
     logged_out: "Successfully logged out."
   notes_controller:
     research_notes: "Research notes"
-    thank_you_for_contribution: "Success! Thank you for contributing open research,
-      and thanks for your patience while your post is approved by <a href='/wiki/moderation'>community
-      moderators</a> and we'll email you when it is published. In the meantime, if
-      you have more to contribute, feel free to do so."
++    thank_you_for_contribution: |
```

Comment:
Hi, thank you for your help, this is causing an error because you added the `+` at the start of the line. You can remove that and just be sure the number of spaces is correct (just like it had been in the old version!). The `+` is to indicate a line you need to add, in "diff" format, and doesn't need to be included. Thanks!